the-collab-lab/tcl-22-smart-shopping-list
# Incorporate Material UI on Welcome page

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

Incorporate Material UI components (Button, TextField) into Welcome page (`Welcome.js`/ `ShareToken.js`)

**_(styling to come!)_**

## Related Issue

#14 

## Acceptance Criteria

- As a user, I want the the app to look professional and welcoming

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓  | :hammer: Refactoring       |

## Updates

### Before

![image](https://user-images.githubusercontent.com/58532369/118747740-8e44ac80-b828-11eb-892c-2a2b9b3379b9.png)


### After


![image](https://user-images.githubusercontent.com/58532369/118902794-6286fc80-b8e4-11eb-8314-5a7d28de3724.png)


## Testing Steps / QA Criteria

To test the feature, use one of the following ways:

**(Make sure to open the page in incognito mode, or delete the token from your local storage in order to be directed to the welcome screen)**

1. Follow the [preview link](https://tcl-22-shopping-list--pr47-cf-material-ui-welco-1fukodsg.web.app/)
2. Launch the app locally from your terminal:
- Pull down this branch with `git pull origin cf-material-ui-welcome` and check that branch out with `git checkout origin cf-material-ui-welcome`
- Run npm start to launch the app in your browser at http://localhost:3000
